### PR TITLE
Always show a topic collection empty state / feedback card

### PR DESCRIPTION
**Pull request by laurieboyes**

### CODE REVIEW

File: server/lib/deepCopy.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export default (value) => JSON.parse(JSON.stringify(value));
```
Comment (laurieboyes) [inline server/lib/deepCopy.js]: Happy for a discussion on whether or not this is evil. My position is that it's not, but maybe I could do with more descriptive module naming so it's clear how the copy is implemented?

File: server/lib/filter-collections.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+import deepCopy from './deepCopy'
 
+// i.e. 3 and an empty one
 const numberRequired = 4;
 
 function resultOrExcess(data, excess) {
 	let result;
 	if(data.length >= numberRequired) {
 		result = data.slice(0, numberRequired);
+		result[3].isEmpty = true;
```
Comment (keirog) [inline server/lib/filter-collections.js]: Sneaky feature addition

File: server/lib/deepCopy.js (hunk 1 above)
Comment (phamann) [inline server/lib/deepCopy.js]: https://facebook.github.io/immutable-js/ :trollface:

File: server/lib/filter-collections.js (hunk 2 above)
Comment (laurieboyes) [inline server/lib/filter-collections.js]: Whoops! ~~Sneaky feature addition~~ Forgetful PR naming

**Pull request merged**